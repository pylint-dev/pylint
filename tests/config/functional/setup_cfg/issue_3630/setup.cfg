# Don't load from tool.pylint, we only support pylint.
[tool.pylint.MASTER]
init-hook='print("I should NOT print in setup.cfg we only parse 'pylint.'")'

# We do load from pylint.
[pylint.MASTER]
load-plugins=pylint_flask

# We don't load options from random sections
[FORMAT]
max-line-length=220
max-module-lines=2001
