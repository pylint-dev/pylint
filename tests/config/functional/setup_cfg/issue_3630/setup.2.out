************* Module Command line or configuration file
Command line or configuration file:1:0: E0013: Plugin 'pylint_flask' is impossible to load, is it installed ? ('No module named 'pylint_flask'') (bad-plugin-value)
